apps/prairielearn/src/ee/lib/lti13.ts

```diff
@@ -768,19 +762,51 @@ export async function updateLti13Scores(
     }),
   );
 
+  const course_staff_data = await selectUsersWithCourseInstanceAccess({
+    course_instance_id: assessment.course_instance_id,
+    minimal_role: 'Student Data Viewer',
+  });
+  const course_staff = new Set(course_staff_data.map((staff) => staff.uid));
+
   const memberships = await Lti13ContextMembership.loadForInstance(instance);
+
+  const token = await getAccessToken(instance.lti13_instance.id);
   const timestamp = new Date();
+  const counts = {
+    success: 0,
+    error: 0,
+    not_sent: 0,
+  };
 
   for (const assessment_instance of assessment_instances) {
     for (const user of assessment_instance.users) {
-      job.info(`ai=${assessment_instance.id}, ${assessment_instance.score_perc}% for ${user.name}`);
+      const ltiUser = memberships.lookup(user);
+      const is_staff = course_staff.has(user.uid);
+
+      // User not found in LTI, reporting only
+      if (ltiUser === null) {
+        job.info(
+          `Not sending grade ${assessment_instance.score_perc.toFixed(2)}% for ${user.uid}.` +
+            ` Could not find ${is_staff ? 'PL course staff' : 'PL student'} ${user.uid}` +
```

Comment:
Let's avoid the use of the "PL" abbreviation in user-facing UI/docs/etc. Ditto below.

Reply:
Addressed in 725e64af39